### PR TITLE
hotfix

### PR DESCRIPTION
**Pull request by Lonely-fr**
Use "bundle install" and "rails db:setup" seeds.rb updated!

### CODE REVIEW

**Review by Laranto-spb — APPROVED** [review]

**Review by alllpim — APPROVED** [review]

File: app/controllers/consignment_controller.rb [hunk 1]
```diff
@@ -5,16 +5,12 @@ class ConsignmentController < ApplicationController
   load_and_authorize_resource
 
   def index
-    consignments = if @ability_lvl == UserRole::ABILITY_SYSTEM
+    consignments = if ability_system?
```
Comment (e-popolamov) [inline app/controllers/consignment_controller.rb:8]: `consignments = ability_system? ?  Consignment.all : @current_user.company.consignments`

File: app/controllers/consignment_controller.rb [hunk 2]
```diff
@@ -23,18 +19,18 @@ def show
     consignment.reports.each do |_report|
```
Comment (e-popolamov) [inline app/controllers/consignment_controller.rb:19]: use single line

File: app/controllers/report_type_controller.rb [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 class ReportTypeController < ApplicationController
   respond_to :json
-  skip_before_action :access_lvl_helper, :ability_lvl_helper
+  skip_before_action :access_helper, :ability_helper
 
   def index
     types = ReportType.all
```
Comment (e-popolamov) [inline app/controllers/report_type_controller.rb:8]: `render json: { ReportTypes: ReportType.all }, status: :ok`

File: app/controllers/reports_controller.rb [hunk 4]
```diff
@@ -55,34 +55,32 @@ def adaptiv_reports_goods(report)
...
     end
   end
 
   def adaptiv_reports_goods_after_place(consignment, report)
     reported_area = 0
-    report.reported_goods.each do |goods|
-      reported_area += goods.reported_quantity.to_i
-    end
+    report.reported_goods.each { |goods| reported_area += goods.reported_quantity.to_i }
     warehouse = Warehouse.find(consignment.warehouse_id)
```
Comment (e-popolamov) [inline app/controllers/reports_controller.rb:76]: combine this two lines

**Comment by e-popolamov** [comment]
Great work! Now it's look better!

**Pull request merged**